code/modules/food_and_drinks/food/snacks_meat.dm

```diff
@@ -185,7 +185,10 @@
 
 /obj/item/reagent_containers/food/snacks/monkeycube/proc/Expand()
 	visible_message("<span class='notice'>[src] expands!</span>")
-	new /mob/living/carbon/monkey(get_turf(src))
+	var/mob/spammer = get_mob_by_key(src.fingerprintslast)
+	var/msg = "Spawned via [src], Last attached Mob: [spammer.real_name] ([spammer.key])"
```

Comment:
no need to declare this variable, just put direct. Also change the message to `Spawned via [src] at [COORD(src)], Last attached mob: [key_name(spammer)].`